src/js/options.js

```diff
@@ -91,7 +91,7 @@ document.addEventListener("DOMContentLoaded", () => {
   browser && browser.storage.local.get(localSettings => {
     Object.entries(localSettings).forEach(([key, value]) => {
       if (!VALID_SETTINGS.includes(key)) return;
-      HTML.setAttribute(key, value);
+      HTML.setAttribute(key, SETTINGS_LIST[key].values ? SETTINGS_LIST[key].values[value] ?? SETTINGS_LIST[key].values[SETTINGS_LIST[key].defaultValue] : value);
```

Comment:
We need to save the key from `values` in `storage.local`, i.e. `system`, not `true system`/`false system`. Hence, we need to convert the key saved in storage to its corresponding value.

Check if `values` exists, if it does:
- check if the value saved in the storage is present in `values`
    - If it does, then set the attribute to `values[value from storage.local]`
    - If it does not, then set the attribute to `defaultValue`. This is a check, this case may happen due to updates to the extension, if we decide to change variable names.